gem/templates/chhaajaa/comments/comment_block.html

```diff
@@ -8,14 +8,6 @@ <h4 class="heading__subheading">{% trans "Comment" %}</h4>
       {% if article.allow_commenting %}
         {% if request.user.is_authenticated %}
           {% render_comment_form for article %}
-          {% comment %}<!--
-          {% else %}
```

Comment:
this else does need need to be here because in chhaa jaa users are not able to see any content before they have logged in, so the user is always authenticated